### PR TITLE
Zomato Detailed

### LINKED ISSUE #110: [API Request] Get one restaurant
For a given restaurant ID, make a GET API using Zomato API in `food` module to given the following output parameters
Make a new class called `ZomatoDetailedResponse` in a new file `zomato_detailed_response.py`

### Input parameters
+ restaurant ID
### Output parameters
+ id
+ name
+ url
+ address
+ latitude
+ longitude
+ average_cost_for_two
+ price_range
+ currency
+ featured_image
+ user_rating_aggregate_rating
+ user_rating_votes
+ is_delivering_now
+ has_table_booking
+ cuisines
+ photo_count
+ phone_numbers 
+ list of photos
  + photo_id
  + photo_url
  + caption
  + likes_count

### PR DESCRIPTION
**Pull request by darthgera123**
# Description

Fetches a single restaurant information. The rest of the information isnt accessible because of Partner access required

Fixes #110 (issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `flake8`
- [ ] `python manage.py test`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: api/modules/food/constants.py [hunk 1]
```diff
@@ -7,5 +7,6 @@
 ORIGINAL_ZOMATO_API = ""
 ZOMATO_API_KEY = os.environ.get("ZOMATO_API", ORIGINAL_ZOMATO_API)
 BASE_URL = "https://developers.zomato.com/api/v2.1/geocode?lat={0}&lon={1}"
+BASE_URL_ID = "https://developers.zomato.com/api/v2.1/restaurant?res_id={0}"
```
Comment (prabhakar267) [inline api/modules/food/constants.py]: Change `BASE_URL` to `https://developers.zomato.com/api/v2.1/` and then have a `list` API and a `restaurant` API

**Review by prabhakar267 — CHANGES_REQUESTED** [review]
Rest of the changes look fine
I will try it on my local system, and will update the comments if anything else comes up 👍

File: api/modules/food/constants.py (hunk 1 above)
Comment (darthgera123) [reply api/modules/food/constants.py]: okay

File: api/modules/food/constants.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 import os
 ORIGINAL_ZOMATO_API = ""
 ZOMATO_API_KEY = os.environ.get("ZOMATO_API", ORIGINAL_ZOMATO_API)
-BASE_URL = "https://developers.zomato.com/api/v2.1/geocode?lat={0}&lon={1}"
+BASE_URL = "geocode?lat={0}&lon={1}"
```
Comment (prabhakar267) [inline api/modules/food/constants.py]: Change this to a valid `BASE_URL`

**Review by prabhakar267 — CHANGES_REQUESTED** [review]

**Pull request merged**